adapters/nextmillennium/nextmillennium.go

```diff
@@ -15,13 +15,21 @@ import (
 type adapter struct {
 	endpoint string
 	nmmFlags []string
+	server   config.Server
 }
 
 type nmExtPrebidStoredRequest struct {
 	ID string `json:"id"`
 }
+
+type serverOverloader struct {
+	ExternalUrl string `json:"externalurl"`
+	GvlID       int    `json:"gvlid"`
+	DataCenter  string `json:"datacenter"`
+}
```

Comment:
Why is this named overloader? What is being overloaded? `request.ext.prebid.server` is populated with the same data in the request.

Reply:
yes, thanks, old name, forgot to correct it